pkg/api/api.proto

```diff
@@ -212,10 +212,21 @@ service Manager {
             get: "/api/Manager/GetSavedModels"
         };
     };
+    /** 
+     * Validate Suggestion Parameters from Study Job.
+     * Suggestion service should return INVALID_ARGUMENT Error when the parameter is invalid
+     */
+     rpc ValidateSuggestionParameters(ValidateSuggestionParametersRequest) returns (ValidateSuggestionParametersReply){
+        option (google.api.http) = {
```

Comment:
I find that ValidateSuggestionParametersReply is used as `return  &api.ValidateSuggestionParametersReply{}` all the time. can we drop ValidateSuggestionParametersReply, alway return nothing?
or you can use below struct to replace current one 
```
message ValidateSuggestionParametersReply {
    bool is_valid 1;
    string msg 2;
 }
```

Reply:
@hougangliu  But we don't need to return anything, like here: 
https://github.com/kubeflow/katib/blob/master/cmd/manager/main.go#L213. 
If parameter is not valid, we return ```INVALID_ARGUMENT``` Error message.